classes/Http/Action/Talk/CreateProcessAction.php

```diff
@@ -177,7 +177,7 @@ public function __invoke(HttpFoundation\Request $request): HttpFoundation\Respon
 
         $url = $this->urlGenerator->generate('dashboard');
 
-        return new HttpFoundation\RedirectResponse($url);
+        return new HttpFoundation\RedirectResponse($url, HttpFoundation\Response::HTTP_CREATED);
```

Comment:
In theory, you can perform a redirect with a `201` status. But the standard says that the location header should point to the created resource. imho, `302` is the better choice here.

I wouldn't use `201` on form processing, only for APIs.